rust/agama-lib/src/software/model/registration.rs

```diff
@@ -29,6 +29,15 @@ pub struct RegistrationParams {
     pub email: String,
 }
 
+/// Addon registration
+#[derive(Clone, Serialize, Deserialize, utoipa::ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct AddonParams {
+    pub id: String,
```

Comment:
Please, describe each field. Those descriptions will be included in the OpenAPI specification.